modules/identity/pages/admin-api-permissions.adoc

```diff
@@ -289,4 +289,4 @@ These are additional REST APIs that you have access to when you are using the Su
 
 === Additional accessible APIs
 * GET
-** platform/license
+** system/information
```

Comment:
@abirembaut am I wrong with this change?

Reply:
:heavy_check_mark: